### ISSUE TITLE
Is the backbone model a base model or instruction tuned model ?

### ISSUE THREAD

**Issue by Mooler0410** [Leooyii/LCEG#1]
Hi!

Really enjoy reading the paper. 

I have a question about the reported results. I'm wondering what variant is used as the backbone model? In the paper, it's mentioned that the backbone is 'LLAMA2 **base**'.  But it seems that the base model's performance on tasks which requires instruction following is abnormally high, such as those in Longbench (table 4). 

Actually, in my own test, I  found that, due the lack of instruction following abilities, the base model's performance is pretty low. 
e.g.  meta-llama/Llama-2-7b-hf  obtains _5.59 on Qasper_, _9.2 on NarrativeQA_ . 

Could you please elaborate more about the used backbone models? Thanks!

**Comment by Leooyii** [comment]
Thank you for your careful reading and feedback. Our backbone model is the 'LLaMA-2-7B base' rather than the 'LLaMA-2-7B chat'. During the evaluation on LongBench, we applied a simple post-processing technique to the predictions of the base model to better evaluate its capabilities:   
[https://github.com/Leooyii/LCEG/blob/2ce672dc42b9b5d3e408e8117126c28caeaafcc0/longbench/eval.py#L79](https://github.com/Leooyii/LCEG/blob/2ce672dc42b9b5d3e408e8117126c28caeaafcc0/longbench/eval.py#L79)
```python
post_process_list = [ "narrativeqa", "qasper", "multifieldqa_en",
                      "hotpotqa", "2wikimqa", "musique",
                      "trec", "triviaqa", "samsum", "lsht",
                      "passage_count", "passage_retrieval_en"]
if dataset in post_process_list:
    prediction = prediction.lstrip('\n').split('\n')[0]
```
We implemented this post-processing for most of the subtasks, and following this procedure, LongBench's evaluations yield appropriate scores for base models.

Best regards,
